src/popup/router/pages/PermissionsSettings.vue

```diff
@@ -85,11 +85,11 @@ export default {
     async changePermission(name, value) {
       let newValue = value;
       if (name === 'transactionSignLimit') {
-        if (Number.isNaN(newValue) || !Number.isFinite(newValue) || newValue < 0) {
```

Comment:
@etharner did you used `Number.isFinite` to disallow typing `Infinity`? I think this is an edge case that we can ignore currently

Reply:
It appears when user is typing very large number in input, but I agree we can omit this check